addons/common/functions/fnc_healUnit.sqf

```diff
@@ -0,0 +1,30 @@
+/*
+ * Author: mharis001, 3Mydlo3
+ * Heals the given unit.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call zen_common_fnc_healUnit
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_unit"];
+
+if (GVAR(aceMedicalLoaded) && {ace_medical_level > 0}) then {
+    ["ace_medical_treatmentAdvanced_fullHealLocal", [_unit, _unit], _unit] call CBA_fnc_targetEvent;
```

Comment:
This will change on next ACE update to
`ace_medical_treatment_fnc_fullHealLocal`
You can detect the next ace update with CfgPatches class "ACE_medical_treatment"
https://github.com/acemod/ACE3/tree/master/addons/medical_treatment

Reply:
I will add a check then, thanks for the info. 👍